### PR TITLE
add OpenSearch support

### PR DESCRIPTION
**Pull request by fiatjaf**

### CODE REVIEW

File: public/opensearch.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/" xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+  <ShortName>cdnjs</ShortName>
+  <Description>Search libraries</Description>
+  <InputEncoding>UTF-8</InputEncoding>
+  <Image width="16" height="16" type="image/x-icon"></Image>
+  <Url type="text/html" method="get" template="https://cdnjs.com/#q={searchTerms}"/>
```
Comment (PeterDaveHello) [inline public/opensearch.xml]: I think this pattern won't work on our search, what do you think @thomasdavis ?

Comment (fiatjaf) [inline public/opensearch.xml]: It will work if the JS code I added over there!

**Comment by PeterDaveHello** [comment]
Okay, let's try it, thanks!

**Comment by PeterDaveHello** [comment]
Hey @fiatjaf, it works very well, thank you very much for your contribution :+1:

**Comment by thomasdavis** [comment]
What exactly is opensearch?

**Comment by PeterDaveHello** [comment]
For browsers' search bar.

**Pull request merged**